### PR TITLE
5083 create api get trade logs by tx hash

### PR DESCRIPTION
**Pull request by NNagato**

### CODE REVIEW

File: tradelogs/storage/influx/influx.go [hunk 1]
```diff
@@ -146,6 +146,11 @@ func prepareTradeLogQuery() string {
 	return tradeLogQuery
 }
 
+// LoadTradeLogsByTxHash get list of tradelogs by tx hash
+func (is *Storage) LoadTradeLogsByTxHash(tx ethereum.Hash) ([]common.TradeLogBasic, error) {
+	return nil, nil
```
Comment (secmask) [inline tradelogs/storage/influx/influx.go]: this should return errors.New("not supported")

Comment (NNagato) [reply tradelogs/storage/influx/influx.go]: updated in https://github.com/KyberNetwork/reserve-stats/commit/c27f8bda057d22db6ac3b6c1d4ddeb11123fdc26

File: tradelogs/common/types.go [hunk 2]
```diff
@@ -64,14 +64,29 @@ type TradeLog struct {
...
-	ETHUSDRate     float64 `json:"-"`
+	ETHUSDRate     float64 `json:"eth_usd_rate"`
 	ETHUSDProvider string  `json:"-"`
 
 	UserName  string `json:"user_name"`
 	ProfileID int64  `json:"profile_id"`
 	Index     uint   `json:"index"` // the index of event log in transaction receipt
 }
 
+// TradeLogBasic tradelog with basic info
```
Comment (secmask) [inline tradelogs/common/types.go]: no use struct?

Comment (NNagato) [reply tradelogs/common/types.go]: removed in https://github.com/KyberNetwork/reserve-stats/pull/409/commits/2d4f78688af67a5b3a9a0f08a34116d69f933de9

**Pull request merged**